# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glib/no-statx.patch.alpha
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- glib-2.66.1/meson.build.vanilla	2020-10-04 14:28:43.665526843 +0200
+++ glib-2.66.1/meson.build	2020-10-04 14:28:52.457526796 +0200
@@ -342,7 +342,7 @@
   }
   '''
 if host_system != 'android' and cc.compiles(statx_code, name : 'statx() test')
-  glib_conf.set('HAVE_STATX', 1)
+  #glib_conf.set('HAVE_STATX', 1)
 endif
 
 if glib_conf.has('HAVE_LOCALE_H')
